### PR TITLE
Allow any ssh urls

### PR DESCRIPTION
**Pull request by chonthu**
Sorry for the delay in pull request, looks like you started on something, but this should cover any ssh urls

### CODE REVIEW

**Comment by chonthu** [comment]
@mattfarina any thoughts on this?

**Comment by mattfarina** [comment]
Sorry for the slow response on this. I'll test it in the next few days. I have not forgotten.

File: vcs_remote_lookup_test.go [hunk 1]
```diff
@@ -29,6 +29,7 @@ func TestVCSLookup(t *testing.T) {
 		"https://gopkg.in/tomb.v1":                                         {work: true, t: Git},
 		"https://golang.org/x/net":                                         {work: true, t: Git},
 		"https://speter.net/go/exp/math/dec/inf":                           {work: true, t: Git},
+		"git@github.com:Masterminds/vcs.git":                               {work: true, t: Git},
```
Comment (dmitris) [inline vcs_remote_lookup_test.go]: could we also add a test with hostname other than github - for example `git@git.example.com:foo/bar.git`, to make sure that the Git VCS detection works correctly not only for github.com but also other git@&lt;hostname&gt; cases?  I would  really appreciate it.  (related to https://github.com/Masterminds/glide/issues/301)

Comment (chonthu) [inline vcs_remote_lookup_test.go]: Yep, ill update to also add bitbucket and custom

**Comment by mattfarina** [comment]
FYI, I'm going to put in some good time on this project next week and try to do another release.

**Comment by chonthu** [comment]
Thanks for the merge! Let me know if I can help with anything else!

**Pull request merged**